drivers/flash/Kconfig.stm32

```diff
@@ -14,7 +14,7 @@ config SOC_FLASH_STM32
 	default y
 	select FLASH_PAGE_LAYOUT
 	select FLASH_HAS_PAGE_LAYOUT
-	select FLASH_HAS_EX_OP if SOC_SERIES_STM32F4X
+	select FLASH_HAS_EX_OP if SOC_SERIES_STM32F4X || SOC_SERIES_STM32L4X
```

Comment:
Please move `select  FLASH_HAS_EX_OP` to `config FLASH_STM32_READOUT_PROTECTION/FLASH_STM32_WRITE_PROTECT`, this should simplify things in the end.

Reply:
done